src/Wallabag/CoreBundle/Resources/config/services.yml

```diff
@@ -220,6 +220,13 @@ services:
             - "%wallabag_core.site_credentials.encryption_key_path%"
             - "@logger"
 
+    wallabag_core.helper.url_hasher:
```

Comment:
I'm wondering if this should be a service or a simple class with static method in it :thinking: 
BTW, the file is missing `UrlHasher.php`

Reply:
:man_facepalming:

Reply:
I thought about a static class, then figured that we may want to be able to change the hash algorithm later on at runtime, so made that a configurable instance attribute.

Reply:
For now I do not think we'll need to update the algorithm at runtime and I prefer to keep that helper dead simple with a static method and no service.

`sha1` can still be the default algo.

```php
<?php

namespace Wallabag\CoreBundle\Helper;

/**
 * Hash URLs for privacy and performance.
 * Internal helper to centralize how the url is hashed.
 */
class UrlHasher
{
    /**
     * @param string $url
     * @param string $algorithm Available algo for the PHP function `hash`
     *
     * @return string hashed $url
     */
    public function hash(string $url, string $algorithm = 'sha1')
    {
        return hash($algorithm, $url);
    }
}
```